Source/UseCases/OnboardingUseCase.swift

```diff
@@ -14,6 +14,10 @@ protocol OnboardingUseCase {
     var hasAcceptedTermsOfService: Bool { get }
     func didAcceptTermsOfService()
 
+    var hasOptedForAnalyticsPermission: Bool { get }
+    func optedInForAnalytics()
```

Comment:
maybe we can have one method: `func optedForAnalytics(acceptsTracking: Bool)` (or maybe you can come up with a better naming than that)? Instead of having two separate methods?

Reply:
One method is much better. Will try a better naming also.

Reply:
This can't be renamed since "hasOptedForAnalyticsPermission" only tells the Coordinator if the user chose something, it doesn't care what. Please take a look.

Reply:
Renamed correctly